packages/web3-eth/src/errors.ts

```diff
@@ -0,0 +1,21 @@
+/* eslint-disable max-classes-per-file */
+
+import { Web3Error } from 'web3-utils';
+
+export class InvalidTransactionWithSender extends Web3Error {
+	// eslint-disable-next-line @typescript-eslint/no-explicit-any
+	public constructor(value: any) {
+		// TODO Discuss this naive approach to logging object
+		// Does not account for non JSON properties
+		super(JSON.stringify(value), 'invalid transaction with sender');
```

Comment:
@nazarhussain Any idea how better to do this? The values passed to these error constructors really is `any` as we're throwing an error because we came across an unexpected value. If a `bigint` was passed here this would throw an internal uncaught exception

Reply:
In such cases if its just the matter of errors and exceptions then we can stick to one format. And when ever we throw error we should use that. e.g. We can say that what ever value will be user will parse to string and then create error constructor. This way it will be consistent to handle.

Reply:
https://github.com/ChainSafe/web3.js/issues/4583